egov-indexer/project-indexer.yml

```diff
@@ -5,8 +5,8 @@ ServiceMaps:
     - topic: save-project-staff-topic
       configKey: INDEX
       indexes:
-        - name: projectStaffIndex-v1
-          type: project-staff
+        - name: projec-staff-index-v1
```

Comment:
missing `t`
should be `project-staff-index-v1`

Reply:
Fixed typo